list1.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LIST1_H
+#define LIST1_H
+#include "list.h"
+class List1
+{
+	private:
+		struct Node
+		{
+			Node* next;
+			int value	
+		};
```

Comment:
no documentation for anything?  the header file is the place to describe how the class works, how it's intended to be used, etc.  See the documentation in list.h for an example.